### DISCUSSION TITLE
How do you know when blenderbot2 uses internet to fetch answers?

### DISCUSSION THREAD

**Discussion by Jawad1347** [facebookresearch/ParlAI#3911] [Ideas]
I have not found satisfactory answers from blenderbot2, some questions were answered surprisingly well, but they could be just out of luck as it said that DMX died at age of 50 and Amelia Clark (mother of dragons) is 34 years old but it did not know what year it was and thinks that Donal Trump is current president. So I am not sure whether the correct answers were fluke or real.

Thank you

**Comment by klshuster** [comment]
if you specify `--loglevel debug` during interactive, you should be able to see when BB2 accesses the internet
